app/services/irs_attempts_api/tracker_events.rb

```diff
@@ -50,6 +50,15 @@ def mfa_enroll_totp(success:)
       )
     end
 
+    # Tracks when the user has attempted to enroll the WebAuthn-Platform MFA method to their account
+    # @param [Boolean] success
```

Comment:
Thinking out loud: We should update the schema doc to describe these new events, if you haven't already.

We might want to be more concrete there -- IIRC, "platform" is something like a laptop's built-in fingerprint reader (touch ID), while the non-platform variety is a removable thing like a Yubikey?

Reply:
Ah - gotcha, I initially thought you were talking about changing the code - but you're talking about being more concrete in the Schema doc. 👍